aws/main.tf

```
@@ -0,0 +1,8 @@
+provider "aws" {
+  region = "us-east-1"
+}
+
+resource "aws_s3_bucket" "my_bucket" {
+  bucket = "my-app-bucket"
+  // Encryption is missing
```

🚩 **Issue Detected**:
The Jira ticket `ABC-1234` specifies that all S3 buckets must have server-side encryption using AES-256. However, encryption is missing in the resource definition
```suggestion
  server_side_encryption_configuration {
    rule {
      apply_server_side_encryption_by_default {
        sse_algorithm = "AES256"
      }
    }
  }
```